src/Microsoft.AspNet.Http/Features/FormFeature.cs

```diff
@@ -152,6 +152,9 @@ public IFormCollection ReadForm()
                 }
             }
 
+            // Rewind so later readers don't have to.
+            _request.Body.Seek(0, SeekOrigin.Begin);
```

Comment:
curious should we check if the request body is seekable?...like one could disable request buffering with `IHttpBufferingFeature`